Filters/CDR0000000154.xml

```diff
@@ -1,10 +1,13 @@
 <?xml version="1.0" ?>
+<!-- File name:    CDR0000000154.xml      -->
 <!-- Filter title: Miscellaneous Document Report Filter -->
+<!-- ================================================== -->
 <xsl:transform               xmlns:xsl = "http://www.w3.org/1999/XSL/Transform"
-                               version = "1.0"
-                             xmlns:cdr = "cips.nci.nih.gov/cdr">
+                             xmlns:cdr = "cips.nci.nih.gov/cdr"
+                               version = "1.0">
```

Comment:
I'm curious—what was behind this change?

Reply:
You will laugh at me but I think it looks better with the shorter line at the bottom.
It's just for consistency thing.  All (maybe only most) filters I worked on list the version last in the "transform" statement.